### ISSUE TITLE
SuperPoint with no GPU

### ISSUE THREAD

**Issue by takahiko-hasegawa-hdjp** [introlab/rtabmap#1176]
HI,

Im trying to use SuperPoint without GPU in a docker environement. 
Is there a way to do that?

thanks in advance

**Comment by matlabbe** [comment]
Yes, you my start from an existing docker image containing pytorch already installed like we did in this one: https://github.com/introlab/rtabmap/blob/master/docker/frontiers2022/Dockerfile

Similarly for if you need the ros version:
https://github.com/introlab/rtabmap_ros/blob/master/docker/noetic/superpoint/Dockerfile

You can launch rtabmap with `SuperPoint/Cuda` parameter set to false to use the CPU.
